### DISCUSSION TITLE
Scalability of Kimai

### DISCUSSION THREAD

**Discussion by Sinns** [kimai/kimai#3416] [On-Premise: Installation & Updates] [answered]
Hey
I was wondering if Kimai can handle 700 users simultaneously. Has anybody some insights? What would the server requirements be?
Best regards

**Comment by kevinpapst** [comment, marked as answer]
Sure, no problem. Maybe not on a 2GB Raspi though...
But some reports might become slow or unusable at some point in time.

**Reply by Sinns** [reply]
Thanks a lot for your quick reply.
